core/views.py

```diff
@@ -4,11 +4,17 @@
 from core.forms import DeckForm, NewCardForm
 from django.contrib.auth.decorators import login_required
 from django.core.paginator import Paginator
+from django.db.models import Q
 
 
 # Create your views here.
 def home(request):
-    decks = Deck.objects.all()
+    decks = Deck.objects.filter()
+    if request.user.is_authenticated:
+        # profile = Profile.objects.get(user__username=request.user.username)
+        decks = Deck.objects.filter((Q(created_by__user=request.user) & Q(ownership=True)) | Q(ownership=False))
```

Comment:
From talks today I think we want to show all public decks on this page. Is this just showing the decks from the logged-in user?